content/en/blog/_posts/Introducing-Kueue.md

```diff
@@ -0,0 +1,207 @@
+- ClusterQueue: a cluster-scoped API to define resource pools by setting quotas for one or more ResourceFlavor. 
+- LocalQueue: a namespaced API for grouping and managing single tenant jobs. In its simplest form, a LocalQueue is a pointer to the ClusterQueue 
+  that the tenant (modeled as a namespace) can use to start their jobs.
+
+For more details, take a look at the [API concepts documentation](https://sigs.k8s.io/kueue/docs/concepts). While the three APIs may look overwhelming, 
+most of Kueue’s operations are centered around ClusterQueue; the ResourceFlavor and LocalQueue APIs are mainly organizational wrappers.
+
+### Example use case
+Imagine the following setup for running batch workloads on a Kubernetes cluster on the cloud: 
+- You have [cluster-autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) installed in the cluster to automatically
+  adjust the size of your cluster.
+- There are two types of autoscaled node groups that differ on their provisioning policies: spot and on-demand. The nodes of each group are 
+  differentiated by the label `instance-type=spot` or `instance-type=ondemand`.
+  Moreover, since not all jobs can tolerate running on spot nodes, the nodes are tainted with `spot=true:NoSchedule`.
+- To strike a balance between cost and resource availability, imagine you want jobs to use up to 1000 cores of on-demand nodes, then use up to
+  2000 cores of spot nodes.
+
+As an admin for the batch system, you define two ResourceFlavors that represent the two types of nodes:
+
+```yaml
+---
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: ResourceFlavor
+metadata:
+  name: ondemand
+  labels:
+    instance-type: ondemand 
+---
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: ResourceFlavor
+metadata:
+  name: spot
+  labels:
+    instance-type: spot
+taints:
+- effect: NoSchedule
+  key: spot
+  value: "true"
+```
+
+Then you define the quotas by creating a ClusterQueue as follows:
+```yaml
+---
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: ClusterQueue
+metadata:
+  name: research-pool
+spec:
+  namespaceSelector: {}
+  resources:
+  - name: "cpu"
+    flavors:
+    - name: ondemand
+      quota:
+        min: 1000
+    - name: spot
+      quota:
+        min: 2000
+```
+
+Note that the order of flavors in the ClusterQueue resources matters: Kueue will attempt to fit jobs in the available quotas according to 
+the order unless the job has an explicit affinity to specific flavors. 
+
+For each namespace, you define a LocalQueue that points to the ClusterQueue above:
+
+```yaml
+---
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: LocalQueue
+metadata:
+  name: training
+  namespace: team-ml
+spec:
+  clusterQueue: research-pool
+```
+
+Admins create the above setup once. Batch users are able to find the queues we are allowed to
+submit to by listing the LocalQueues in their namespace(s). The command is similar to the following: `kubectl get -n my-namespace localqueues`
```

Comment:
In this paragraph, "we are" sounds vague, because it's not clear who "we" refers to.
Would it be better to say 
"Batch users are able to find the queues that Kueue is allowed to

Reply:
it should be "they are allowed to"